content/doc/book/using/using-agents.adoc

```diff
@@ -196,3 +196,26 @@ Building remotely on agent1 in workspace /home/jenkins/workspace/First Job to Ag
 agent1
 Finished: SUCCESS
 ----
+
+=== Launching agent from console
+
+agent.jar is meant to be launched by Jenkins, typically through a remote
```

Comment:
is this actually accurate though? inbound agents are a big thing, and they are invoked with `java -jar agent.jar` + other arguments